components/Ref/Ref.tsx

```diff
@@ -46,7 +46,7 @@ function Ref(props: Props) {
             <hr />
             <div>
                 <Card style={{ marginBottom: '15px' }}>
-                    <Card.Header>Referral system</Card.Header>
+                    <Card.Header><h2>Referral System</h2></Card.Header>
```

Comment:
Dont put a `<h2>` tag distupting the bootstrap styling.
Use the <Card.Title> that is used on most other places and was forgotten here